addon/helpers/object.js

```diff
@@ -1,9 +1,17 @@
 import { MicroState } from 'ember-microstates';
 import assign from '../utils/assign';
 import { reduceObject } from '../utils/object-utils';
+import isPrimitive from '../utils/is-primitive';
 
 export default MicroState.extend({
 
+  initialValueFor([object = {}]) {
+    if (isPrimitive(object) || object == null) {
```

Comment:
Isn't `null` considered a primitive value? I'd just add `null` to the definition and have `isPrimitive(null)` return true.